Sprint-2/debug/0.js

```diff
@@ -1,7 +1,22 @@
 // Predict and explain first...
 
+/*it will  print 
+the first console.log (a * b) which will call the function inputs 10, 32 = 320
 function multiply(a, b) {
   console.log(a * b);
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+
+DEBUG*/
+
+//The result of multiplying 10 and 32 is 320
```

Comment:
Did this prediction bear out, or did it print undefined? Why did it do this?

Reply:
The function wasn't called initially, so it didn't display the result. However, it has now been called and is returning the result.